### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by wenchenglu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+# Release 1.8.0 12-09-2017
+
+- add a test case for OPTIONS. (#261)
```
Comment (qiwzhang) [inline CHANGELOG.md]: we only list of bug fix and new features in the CHANGE log.  not internal changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+
+- Upgrade nginx to 1.13.4 (#242)
+
+- Remove unused stub_map (#206)
+
+- add permission to cp (#253)
+
+- Adding ES256 for auth jwt validator (#231)
+
+- Used ngx_worker to access shared memory for stats (#252)
```
Comment (qiwzhang) [inline CHANGELOG.md]: change this to:  support Nginx restart

File: CHANGELOG.md (hunk 1 above)
Comment (wenchenglu) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+- e2e test service configuration supports x-google-management (#259)
+- Auto detect custom verb to support : in the path. (#257)
+- Upgrade nginx to 1.13.4 (#242)
+- Remove unused stub_map (#206)
+- Adding ES256 for auth jwt validator (#231)
+- Support Nginx restart (#252)
+- Add IAP header support (#251)
+- Adding authz cache. (#225)
+
```
Comment (qiwzhang) [inline CHANGELOG.md]: remove line 3, and 6, please

Comment (wenchenglu) [reply CHANGELOG.md]: done

**Pull request merged**